docs/blog/2025-Q3-community-board.rst

```diff
@@ -0,0 +1,63 @@
+
+Q3-2025 quarterly meeting highlights
+------------------------------------
+
+Our third meeting of the year covered the latest updates for the Berlin conference, as well as general team updates and efforts to improve the organizational structure of the team Slack channels. 
+
+Here are the highlights from the meeting:
+
+**WTD Berlin updates**
+    Now that the CFP is open and tickets are on sale for our European conference in Berlin, we're grateful to have Katie Janovec, our much-loved event producer for the Portland conference, take over the executive coordination and provide support to the conference chair Sasha Romijn. 
+
+    Katie is already hard at work building the graphic resources, production schedule, and marketing assets for the conference. She will be working closely with Sasha as well as with Sam Wright on speaker coordination and volunteers, and we look forward to see her in Berlin to celebrate the event with us. 
+    
+    The conference tickets are selling fast, and we're thrilled to be able to offer a virtual option this year, similar to Portland. This will enable anyone to attend the conference virtually in case they are unable to travel in person. Get your tickets and find out more information by visiting the `Write the Docs Berlin 2025 <https://www.writethedocs.org/conf/berlin/2025/>`_ website. 
+
+**Slack channel organization** 
+    Eric Holscher is driving an effort to improve the organizational structure of our team-focused Slack channels. In addition to providing our community with public channels to interact, our Slack workspace also serves as a place for the community teams to coordinate activities and discuss pertinent topics using private channels for active team members.  
+
+    In the past we've maintained a "staff" channel that included all of the organizational team members. Over time, the membership in the channel got stale and we have been discussing the value of such a broad distribution list in a single channel. The board ultimately decided that since each conference and community team already has their own channel for members and team leads, and the community board has a dedicated channel for all team leads, there isn't a need anymore for an all-hands style space. 
+
+    Each team lead will continue to manage membership in their team channels, and yours truly will continue to manage the community board channel. If we see a demand in the future to revive the staff channel or a similar space for more broad instant communication, we will discuss it with the team leads.
+
+**Meetups updates**
+    Virtual meetups now have an updated 2FA solution with 1Password, and Rose Williams reported that it is working well and much improved over the previous 2FA solutions, which required manual intervention much more often. This update supports smoother meetup organizer workflow when launching virtual meetups, and Rose will continue to seek feedback from the meetup organizers in case there are any issues. 
```

Comment:
```suggestion
    Virtual meetups now have an updated 2FA solution, and Rose Williams reported that it is working well and much improved over the previous workflow, which required manual intervention much more often. This update supports smoother meetup organizer workflow when launching virtual meetups, and Rose will continue to seek feedback from the meetup organizers in case there are any issues. 
```